docs/ubuntu.md

```diff
@@ -78,13 +80,14 @@ Ok, now we are ready to start.
 
 ## Step 1: Deployment
 
-##### Prefer to watch a video of how to install? Go to our [`user contributed documentation`](https://github.com/esmero/archipelago-deployment/blob/1.0.0-RC1/docs/ubuntu.md#user-contributed-documentation)!
+##### Prefer to watch a video of how to install? Go to our [`user contributed documentation`](#user-contributed-documentation-a-video)!
```

Comment:
@aksm this video might not be updated enough anymore.. @alliomeria what do you think?

Reply:
@DiegoPino & @aksm think some might still find referencing this older video to be helpful context. Maybe update the description to something like "You may find this 'user contributed tutorial video' which was created for an earlier Archipelago release to be helpful. Please note there are significant differences between the executed steps, and that you need to follow the current release instructions in order to have a successful deployment." Too much? Better to remove for now & add a reference back later on for an updated video?

Reply:
Ok, that sounds good. But be sure its not a "1:1" doc for this release.@aksm can you make the change?
We also need the M1 docker-compose, will submit in 10/15 minutes, need to fetch lunch or something and be back

Reply:
@DiegoPino @alliomeria ok, added a footnote. Let me know if you think that works or doesn't.